apps/openmw/mwgui/charactercreation.cpp

```diff
@@ -87,6 +87,10 @@ namespace MWGui
     CharacterCreation::CharacterCreation(osg::Group* parent, Resource::ResourceSystem* resourceSystem)
         : mParent(parent)
         , mResourceSystem(resourceSystem)
+        , mPlayerMajorSkills()
+        , mPlayerMinorSkills()
+        , mPlayerAttributes()
+        , mPlayerSkillValues()
```

Comment:
These members are class-based (std::map, std::vector). Do they actually need to be default-initialized? I'm not sure.

Reply:
More likely they are not. I just took an original initialization from WindowManager.